ci/gpuci/run.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+set -e
+
+# Overwrite HOME to WORKSPACE
+export HOME=$WORKSPACE
+
+# Install gpuCI tools
+curl -s https://raw.githubusercontent.com/rapidsai/gpuci-tools/master/install.sh | bash
+source ~/.bashrc
+cd ~
+
+# Show env
+gpuci_logger "Exposing current environment..."
+env
+
+# Login to docker
+gpuci_logger "Logging into Docker..."
+echo $DH_TOKEN | docker login --username $DH_USER --password-stdin
+
+# Install Jinja deps
+gpuci_logger "Install Jinja and PyYAML..."
+python3 --version
+python3 -m pip install --user -r requirements.txt
+
+# Generate dockerfiles
+gpuci_logger "Generating dockerfiles..."
+python3 generate_dockerfiles.py
```

Comment:
I'm not opposed to leaving this here, but I had removed `python3 generate_dockerfiles.py` from the jobs' shell scripts in Jenkins since it's assumed that the repo files will be up-to-date now.

I had also removed `python3 -m pip install --user -r requirements.txt` since Jinja2/PyYAML are in the AMIs now also.